06_puppy/runnerdave/syncStore.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
Seems to me most go package and filenames are in lowercase only, feels a bit weird with `syncStore.go`

Reply:
Good point, updated